src/app/components/source/SourceComponent.js

```diff
@@ -1054,7 +1053,7 @@ class SourceComponent extends Component {
 
           { !isEditing ?
             <div>
-              { this.state.showTab === 'annotation' ? <Annotations annotations={source.annotations.edges.slice().reverse()} annotated={source} annotatedType="Source" /> : null }
+              { this.state.showTab === 'annotation' ? <Annotations annotations={source.log.edges} annotated={this.props.source} annotatedType="ProjectSource" /> : null }
```

Comment:
*annotations* prop is pointing to a `source` while  *annotated* is pointing to the `project_source` `(this.props.source)`